src/urllib3/request.py

```diff
@@ -65,6 +65,7 @@ def request(
         self,
         method: str,
         url: str,
+        *,
```

Comment:
We can't change this one unfortunately, backwards compatibility needs to be maintained for existing APIs.

Reply:
Ok, I forgot about the backward compatibility... removing it

Reply:
Done